README.md

```diff
@@ -61,6 +68,18 @@ The release process is building all runtimes and then puts them into a release i
 
 The format of [`CHANGELOG.md`](CHANGELOG.md) is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+### Point releases
+
+In order to release a patch or fix outside the normal release process, some additional steps are needed.
```

Comment:
Not ultra happy with this process, but until someone has automated it, it is good to have it written down.

Reply:
Ideally there would be a bot command so you could call `/cmd point-release` or whatever in a comment on a PR made against the `x.y.z` branch, the bot would create branch `x.y.b`, rebase the PR to `x.y.b` then trigger the release process from `release-*` branch when it sees the usual changelog changes when the branch is merged.

Probably would need to know which release is on chain and only allow +1s to avoid lots of branches being made.